pyproject.toml

```diff
@@ -10,10 +10,10 @@ homepage = "https://github.com/python-pinot-dbapi/pinot-dbapi"
 
 [tool.poetry.dependencies]
 python = ">=3.7,<4"
-ciso8601 = "^2.2.0"
+ciso8601 = "^2.1.3"
```

Comment:
question: this means exactly 2.1.3 or higher than or equal to 2.1.3? how does it work?

Reply:
Ref: https://python-poetry.org/docs/dependency-specification/

Reply:
That means any version >= 2.1.3, but lower than 3.0.0.

Reply:
thanks for the answers, i think my original question is more on install time / env.
1. will it automatically pull 2.1.3, or the newest at the time of install? 
2. if a pinot-dbapi with 2.2.0 installation exist and we upgrade, will we reuse the 2.2.0 or the answer from question 1?
3. if another dependency pins the version to 2.2.0 within the same python venv, does it attempt to install 2.1.3? does it keep 2 versions and symlink them differently?

sorry for the confusions :-/

Reply:
No worries at all, man, those questions are an opportunity for everyone to learn more about that tool - including myself! :-)

So, the dependencies in the `pyproject.toml` file are not pinned dependencies, they just define version ranges for the dependencies. The pinned dependencies, however, are defined in the `poetry.lock` file instead. And it seems like this PR missed updating the lock file (with command `$ poetry lock`). So, for example, you might end up seeing, in the lock file, something like version `2.3.4`, but `^2.1.3` in `pyproject.toml` (because it only defines a loose dependency, not a pinned one).